### ISSUE TITLE
due --run environment

### ISSUE THREAD

**Issue by stappersg** [CumulusNetworks/DUE#10]
FWIW    it feels strange to pick an environment from a menu.

Or expressed that feeling somewhat stronger
> selecting a environment from a menu breaks  command line workflow

Please make it possible to  select  environment  as parameter of  _run_

**Comment by ehdoyle** [comment]
The --run-image option under the --run option will allow this ( actually it filters and if there is only 1 entry left, it runs that, otherwise it creates a smaller choice menu )
I think this makes logical sense, but from a user interface standpoint, I think it is less than clear, as I've found myself forgetting exactly what the syntax is, and I wrote it,  so it's a valid complaint.
I was thinking I'd edit the top of the menu to read;

 echo "- Select one of the following containers to run:"              
 echo "-  (Skip this menu by using '--run --run-image <image name>')"   <-new text

So the initial user experience is seeing those options, and then it gives them a way to just run the container from the command line.
Thoughts?
